benchmark/kernels/fused_moe_triton/tuning_fused_moe_triton.py

```diff
@@ -400,6 +400,9 @@ def main(args: argparse.Namespace):
         shard_intermediate_size = 2 * intermediate_size // args.tp_size
     elif config.architectures[0] in ["DeepseekV2ForCausalLM", "DeepseekV3ForCausalLM"]:
         E = config.n_routed_experts
+        n_share_fusion_experts = int(os.getenv("SHARE_EXPERTS_FUSION_REPLICA", "0"))
+        if n_share_fusion_experts > 0:
```

Comment:
nit: maybe we can unify the logic as `E = config.n_routed_experts + n_shared_fusion_experts`

Reply:
ok